O/OptFrame/build_tarballs.jl

```diff
@@ -0,0 +1,48 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "OptFrame"
+version = v"5.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/OptFrame/OptFrame.git", "d30011695eac4532f9592a17f91b39e9d44f0069")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/OptFrame/
+
+make optframe_lib
+
+mkdir -p ${libdir}
+
+cp ${WORKSPACE}/srcdir/OptFrame/build/optframe_lib.so ${libdir}/optframe_lib.${dlext}
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = expand_cxxstring_abis([
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("powerpc64le", "linux"; libc = "glibc"),
+    Platform("riscv64", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("aarch64", "linux"; libc = "musl"),
+    Platform("x86_64", "freebsd"; ),
+    Platform("aarch64", "freebsd"; ),
+    Platform("i686", "windows"; ),
+    Platform("x86_64", "windows"; )
```

Comment:
What's the problem with the other platforms?

Reply:
Hi @giordano, thanks for your help :) There were some compilation issues with the other plats, but they are pretty much fixed and I will be adding them back with the next tag.